test/gpdb/run_docker.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+DOCKERIMAGE="$1"
+LOG_DIR="$2"
+SCRIPT_PATH=$(dirname $(realpath -s $0))
+PROJECT_ROOT=$(dirname $(dirname "$SCRIPT_PATH"))
+
+docker build -t pgbackrest:test -f $SCRIPT_PATH/Dockerfile \
+$PROJECT_ROOT --build-arg GPDB_IMAGE=$DOCKERIMAGE
+
+
+if [ -d "$LOG_DIR" ]; then
+	docker run --rm  \
```

Comment:
replacing tabs with spaces.
```suggestion
    docker run --rm  \
```